Data/Mu2e2Tracker_IT1-2s.dat

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
I suggest these files to go in a dedicated directory, otherwise the Data folder can become messy

Reply:
Yes, not all them should be imported at the end. In the mean time I'll move them in a separate folder